app/controllers/providers/means/capital_disregards/add_details_controller.rb

```diff
@@ -0,0 +1,43 @@
+module Providers
+  module Means
+    module CapitalDisregards
+      class AddDetailsController < ProviderBaseController
+        prefix_step_with :capital_disregards
+        def show
+          @form = AddDetailsForm.new(model: capital_disregard)
+        end
+
+        def update
+          @form = AddDetailsForm.new(form_params)
```

Comment:
Should this meet the current pattern, that is to chain the module definitions `Providers::Means::CapitalDisregards::AddDetailsForm`?